DashClothes/urls.py

```diff
@@ -19,5 +19,10 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('accounts/', include('allauth.urls')),
+    path('app/',include('accounts.urls')),
```

Comment:
you can give an empty path for all auth URLs. 
why give an "app" URL path for custom project app URLs? i think it's a bit confusing. @Yshah112
cc. @lalitvasoya @rinkishivani

Reply:
-change allauth  url to blank url 
-changed application url name to accounts